### DISCUSSION TITLE
Cant see assets when deployed

### DISCUSSION THREAD

**Discussion by lila-papazian** [vercel/community#2099] [Help]
### Summary

Hi!
I'm working with Astro doing some static pages.
All my assets are in the public folder, and called <img src="public/asset1" ...
Works well on local but after deploy using Vercel I can't see any of them.
What am I missing?
TIA

### Example

https://san-quirino-estudio-8l5izqavx-lila-papazian.vercel.app/

**Comment by swarnava** [comment]
Hi,

It looks like they are being fetched from a different directory which doesn't exist:

<img width="409" alt="image" src="https://user-images.githubusercontent.com/1162991/232559849-28bccd9d-d2e0-4e0e-b726-e87b0a8ffdca.png">

**Reply by lila-papazian** [reply]
Actually, `public` does exist as a folder that contains the SVG. I don't know how the URL is built tho. Is there some documentation that explains that?
The assets are called like this `<img src="public/sanquirino-logo.svg" />`, but it only works on local
